contracts/modules/TransferManager.sol

```diff
@@ -502,25 +461,6 @@ contract TransferManager is BaseModule, RelayerModule, LimitManager {
         return true;
     }
 
-    // Overrides to use the incremental nonce and save some gas
-    function checkAndUpdateUniqueness(BaseWallet _wallet, uint256 _nonce, bytes32 /* _signHash */) internal returns (bool) {
-        return checkAndUpdateNonce(_wallet, _nonce);
-    }
-
-    function validateSignatures(
-        BaseWallet _wallet,
-        bytes memory /* _data */,
-        bytes32 _signHash,
-        bytes memory _signatures
-    )
-        internal
-        view
-        returns (bool)
-    {
-        address signer = recoverSigner(_signHash, _signatures, 0);
-        return isOwner(_wallet, signer); // "TT: signer must be owner"
-    }
-
     function getRequiredSignatures(BaseWallet /* _wallet */, bytes memory _data) internal view returns (uint256) {
```

Comment:
We should sync with @petar-argent and use the default implementation provided by `OnlyOwnerModule`.

Reply:
Yes, will be updating the three contracts that need to be simplified once the backend is ready.

@petar-argent for reference the full list of methods that it'd be great for the backend to call directly instead of via execute is:

GuardianManager:
confirmGuardianAddition(address,address)
confirmGuardianRevokation(address,address)

RecoveryManager:
finalizeRecovery(address)

TransferManager:
executePendingTransfer(address,address,address,uint256,bytes,uint256)"

Reply:
Adding @dappcoder to this thread